### PR TITLE
Add is_paused param to schedules

### PR DESCRIPTION
**Pull request by emelvarol-cognite**

### CODE REVIEW

File: cognite/transformations_cli/commands/deploy/transformation_types.py [hunk 1]
```diff
@@ -69,7 +75,7 @@ class TransformationConfig:
     name: str
     query: Union[str, QueryConfig]
     authentication: Union[AuthConfig, ReadWriteAuthentication]
-    schedule: Optional[str]
+    schedule: Optional[Union[str, ScheduleConfig]]
```
Comment (hakontro) [inline cognite/transformations_cli/commands/deploy/transformation_types.py:78]: Can we change ScheduleConfig to avoid the Union here?

Comment (emelvarol-cognite) [reply cognite/transformations_cli/commands/deploy/transformation_types.py:78]: It will break backward compatibility in case anyone started using this.

Comment (emelvarol-cognite) [reply cognite/transformations_cli/commands/deploy/transformation_types.py:78]: if user doesn't need pause option it will be `schedule: * * * * *`

**Comment by emelvarol-cognite** [comment]
👍

**Review by hakontro — APPROVED** [review]
LGTM

**Pull request merged**